### ISSUE TITLE
User auth

### ISSUE THREAD

**Issue by pepopowitz** [CentareGroup/mkedotnet.com#6]
My personal recommendation is to use something like Firebase to manage user auth, but I'm open to other ideas. I do think this can be handled by a 3rd-party service to manage user auth, if it isn't Firebase.

It'd be nice for users to be able to log into the site. Once we can do that, we can do fun things like "build my schedule", and "submit a rating of a talk."

**Comment by Zuul86** [comment]
Is that this?
https://firebase.google.com/products/auth/

**Comment by pepopowitz** [comment]
Yeah, that looks right! I would imagine it's the same thing that we used to build the Centare "device manager", except the source of the users could be whatever we decide, instead of coming from Centare SAML. 

I'm sure there are other providers we could use, too. Auth0 comes to mind, but I don't know much about it. Firebase is what first popped into my head, because I think we'd want to use something like it to store the "my sessions" & session rating information, along side the auth info.

**Comment by Zuul86** [comment]
Looks like it's supports the major identity providers.  I'll have a look.

**Comment by pepopowitz** [comment]
We might want to keep this stuff in a new branch, off of master - so that any archiving of 2017 (and thus removal of the 2017 schedule) doesn't interfere with it.
